packages/manager/modules/pci/src/projects/project/data-processing/submit-job/spark-sizing/spark-sizing.controller.js

```diff
@@ -0,0 +1,117 @@
+import {
+  MEMORY_OVERHEAD_RATIO,
+  MIN_MEMORY_OVERHEAD_MB,
+} from './spark-sizing.constants';
+
+export default class {
+  /* @ngInject */
+  constructor() {
+    // let's do some bindings
```

Comment:
This comments describes `what you do`, not `why you do something`, please keep your comments for `why you do something`